Commerce packaging fixes (#1313)
ezsystems/developer-documentation: docs/guide/shop_features/product_management.md

# Product management

## Product

[[= product_name =]] offers a built-in [Product Content Type](../catalog/product.md).

Any content of this type can be automatically used for all shop functionalities,
such as adding to basket, price calculation and ordering,

> and can be embedded on the front end with quick ordering buttons.

Reviewer: ```suggestion
and can be embedded on the front end with quick ordering buttons.
```
Does "with quick ordering buttons" mean "together with..."?

> and can be embedded on the front end with quick ordering buttons.

Author: ```suggestion
and can be embedded on the front end together with quick ordering buttons.
```

## Product variants

> A product can have different variants, corresponding for example to different colors or sizes.

Reviewer: ```suggestion
A product can have different variants that correspond, for example, to different colors or sizes.
```

You can select and preview variants on the product's page.
Variants can have one or two levels.

> When you choose a first attribute, the shop narrows down the options for the second level.

Reviewer: ```suggestion
When you choose the first attribute, the shop narrows down the options for the second level.
```
Could it be:
"When you choose an option for the attribute at the first level, the shop narrows down the options for the second level."
?

![Product detail](img/product_detail.png)

## Product type

A product type represents a collection of very similar products that differ only in some characteristics.

> It is used to show a list of products in a tabular way, every product can be added to the basket directly from the overview page. 

Reviewer: ```suggestion
It is used to show a list of products in a tabular way, and every product can be added to the basket directly from the overview page. 
```
